package.json

```diff
@@ -2,34 +2,69 @@
-	"main": "./dist-node/index.js",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain",
+		"nodejs",
+		"javascript",
+		"library"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-js#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-js/issues"
+	},
+	"engines": {
+		"node": "=6.12.3",
+		"npm": "=3.10.10"
+	},
+	"main": "dist-node/index.js",
 	"scripts": {
 		"prestart": "./bin/prestart.sh",
 		"start": "./bin/start.sh",
-		"test": "nyc mocha test --recursive",
-		"test:watch": "chokidar \"./(src|test)/**/*.js\" -c \"mocha test --recursive\" --initial",
+		"clean": "rm -rf ./node_modules ./dist-browser/lisk-js.js ./dist-browser/lisk-js.min.js ./dist-node",
+		"format": "prettier --write \"browsertest/{setup,runTests}.js\" \"{cypress/integration,src,test}/**/*.js\"",
+		"lint": "eslint .",
+		"lint:fix": "npm run lint -- --fix",
+		"test": "NODE_ENV=test nyc mocha test",
+		"test:watch": "npm test -- --watch",
+		"test:watch:min": "npm run test:watch -- --reporter=min",
 		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
-		"cover": "nyc report --reporter=lcov",
-		"eslint": "grunt eslint",
-		"prettier": "prettier --write \"browsertest/{setup,runTests}.js\" \"{cypress/integration,src,test}/**/*.js\"",
+		"cover": "if [ -z $JENKINS_HOME ]; then npm run cover:local; else npm run cover:ci; fi",
+		"cover:base": "NODE_ENV=test nyc report",
+		"cover:local": "npm run cover:base -- --reporter=html --reporter=text",
+		"cover:ci": "npm run cover:base -- --reporter=text-lcov | coveralls -v",
 		"serve:browsertest": "http-server browsertest",
-		"jenkins": "grunt jenkins --verbose",
-		"build": "grunt build",
+		"prebuild:node": "rm -r dist-node/* || mkdir dist-node | echo",
+		"prebuild:browser": "rm ./dist-browser/lisk-js.js ./dist-browser/lisk-js.min.js",
+		"prebuild:browsertest": "cp package.json browsertest/",
+		"prebuild": "npm run prebuild:node && npm run prebuild:browser",
+		"build": "npm run babel && npm run browserify && npm run uglify",
+		"build:browsertest":
+			"npm run babel:browsertest && npm run browserify:browsertest && npm run uglify:browsertest",
+		"build:node": "npm run babel",
+		"build:browser": "npm run babel && npm run browserify && npm run uglify",
 		"build:check": "node -e \"require('./dist-node')\"",
-		"build:browsertest": "grunt build-browsertest",
-		"precommit": "lint-staged && npm run eslint",
-		"prepush": "npm run eslint && npm test",
-		"prepublishOnly": "npm run prepush && npm run build",
+		"babel": "babel src -d dist-node",
+		"babel:browsertest":
+			"babel src -d ./browsertest/src && BABEL_ENV=browsertest babel test --ignore test/transactions/dapp.js -d ./browsertest/test",
+		"postbuild:browsertest": "rm -r browsertest/src browsertest/test",
```

Comment:
Would be great to have a slightly more logical/consistent order for these scripts.